src/app/[locale]/dataset-list.cy.tsx

```diff
@@ -70,3 +70,174 @@ describe('List rendering', () => {
     });
   });
 });
+
+describe('Dataset list filters', () => {
+  beforeEach(function () {
+    cy.fixture('search-results/three-datasets.json').then(searchResult => {
+      this.searchResult = searchResult;
+      cy.intercept({method: 'GET', url: '/api/datasets?*'}, searchResult);
+      cy.mount(
+        <DatasetListWithTranslation
+          initialSearchResult={searchResult}
+          locale="en"
+        />
+      );
+    });
+  });
+
+  it('can filter by one license', function () {
+    cy.intercept(
+      {
+        method: 'GET',
+        url: `/api/datasets?licenses=${encodeURIComponent(
```

Comment:
Rather than preparing the query string manually, would it be an idea to use `URLSearchParams` instead?